### DISCUSSION TITLE
Is it possible to provide defaults to a user?

### DISCUSSION THREAD

**Discussion by volker101** [kimai/kimai#3439] [Questions & Answers] [answered]
Hello all,

I am currently evaluating Kimai for a customer of mine. I couldn't answer his main question: Is it possible to provide defaults to a user for a time tracking project? For example this is the start sheet which requires the user to choose its values

![2022-07-25_18-48](https://user-images.githubusercontent.com/33600023/180832465-a59db6e3-05d5-4b9d-9fda-09268ca7bc9d.png)

Now, I am looking for way that the main fields are filled with default values, instead of the empty lines. Is that possible to achieve?

Thanks,
Volker

**Comment by kevinpapst** [comment, marked as answer]
There are dozens of discussions regarding that topic.
Your customer is welcome to sponsor the development of a plugin.
Otherwise there are multiple ways to start a timesheet

**Comment by volker101** [comment]
Thank you for your answer!

**Reply by kevinpapst** [reply]
Not sure if you understood it though. I am sometimes annoyed by answering the same question over and over again, not your fault obviously! This feature is simply not necessary from my pov.

There is a plugin called "Custom content" and you can use it to inject javascript. And then there are examples to pre-select e.g. an activity, see https://www.kimai.org/documentation/plugin-custom-content.html#javascript

There are multiple ways to start records / create them afterwards and in most cases you will not have to choose project and activity - e.g. by drag & drop in the calendar or the restart dropdown.

**Reply by volker101** [reply]
Sure I can understand you. As a developer live is often not easy.
Thanks again for the additional information.
